src/status_im/contexts/preview/quo/community/channel_action.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im.contexts.preview.quo.community.channel-action
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im.contexts.preview.quo.preview :as preview]))
+
+(def descriptor
+  [{:key  :label
+    :type :text}
+   {:key  :big?
+    :type :boolean}
+   {:key  :counter-value
+    :type :number}
+   {:key     :icon
+    :type    :select
+    :options [{:key :i/muted}
+              {:key :i/pin}]}
+   (preview/customization-color-option {:key :color})
+   {:key  :disabled?
+    :type :boolean}])
+
+(defn view
+  []
+  (let [state (reagent/atom {:label         "Action"
+                             :icon          :i/muted
+                             :color         :blue
+                             :counter-value nil
+                             :disabled?     false
+                             :big?          true})]
+    (fn []
+      [preview/preview-container {:state state :descriptor descriptor}
+       [rn/view {:style {:flex 1 :flex-direction :row}}
```

Comment:
current implementation for channel-action is wrong, there shouldn't be any `[rn/view {:style {:flex 1 :flex-direction :row}}` container to make component work properly, see here i have already fixed that https://github.com/status-im/status-mobile/pull/19032/files#diff-37c5c05d51d3ac0ba6b52c73251d65f42334c528ad3956e3ee6290d46df594c5
https://github.com/status-im/status-mobile/pull/19032/files#diff-78028f7d9a49eba44bc1fe0cb1fe64d62267c67faadfa15bb5b2ddc1b97bddd0

Reply:
thanks, fixed :+1: